CONTRIBUTING.md

```diff
@@ -4,5 +4,5 @@ PyBee <3's contributions!
 
 Please be aware, PyBee operates under a Code of Conduct. 
 
-See [CONTRIBUTING to PyBee](pybee.org/contributing) for details.
+See [CONTRIBUTING to PyBee](pybee.org/contributing/index.html) for details.
```

Comment:
I have made the changes accordingly
As for the question, I follow you on Github. So if you do any activity like making a repo, I come to know about it through the feed.